### DISCUSSION TITLE
(sorry stupid question, amazing work keep it up !) VIRUSTOTAL FALSE POSITIVE?

### DISCUSSION THREAD

**Discussion by LimitedWolf1** [diasurgical/DevilutionX#5310] [Q&A] [answered]
Hi just a question about devilutionX, on virustotal I think it gives a false positive, has this always been the case , its "
Trojan.Malware.300983.susgen". I love the work you all have done I just have bad anxiety and would like to know if all is safe , and why it is appearing this way, sorry I am new to all these sorts of things, thank you!
![vt img](https://user-images.githubusercontent.com/112410077/187231645-3ead86a6-a3b1-41ef-bd28-1d2ebb1ad372.png)

**Comment by StephenCWills** [comment]
Here is the actual link for anyone wanting to review it.
https://www.virustotal.com/gui/file/a105fd56e595011e4b0a6537b6f7123b220fdb32158cc404b9613dc01316dd8e

It's very likely a false positive, based on this account of an entirely different zip file.
https://www.reddit.com/r/techsupport/comments/o52rq3/trojanmalware300983susgen/

**Reply by LimitedWolf1** [reply]
Thank you !

**Comment by AJenbo** [comment, marked as answer]
Many anti virus systems use heuristics. They might be looking at headers, sections of code that captures the mouse, or even just the file size.

The best thing you can do to avoid getting flaged as a virus is paying for a digital signature ($287/Year). But since this is just a hobby that's a bit out of range for us.

**Reply by AJenbo** [reply]
"susgen" likely just there scanners that findes **gen**eral **sus**picion, and not an actual virus identification.

Googleing a bit also indicates that there are many such false positives from MaxSecure.

**Reply by LimitedWolf1** [reply]
Thank you very much for taking the time out to explain this to me, This is a wonderful project you have all worked on and I also thank you for giving me the opportunity to be able to play this masterpiece online, i wasn't even born when this came out but always wanted to play it. This is honestly amazing for a passion project ! Keep up the good work, all of you!!!

**Comment by DakkJaniels** [comment]
> would like to know if all is safe

If the maintainers did put in a virus that no one else picked up, do you think they would respond truthfully? 😂 

(just to clarify, I don't think there is a virus - just think it's funny asking the people who, hypothetically, would have put in a virus if there is one)

**Reply by LimitedWolf1** [reply]
Fair point lmaoo

**Reply by AJenbo** [reply]
To be fair we could have gotten infected an unknowingly spreading a virus :)
